build-helpers/gtk-common-themes-parts.yaml

```diff
@@ -23,6 +34,7 @@ parts:
       set -eu
       [ -d squashfs-root/ ] || exit 1
       cp -a squashfs-root/* $SNAPCRAFT_PART_INSTALL/
-      # remove everything related to communitheme
+      # remove everything related to communitheme and yaru
       rm -rf $SNAPCRAFT_PART_INSTALL/share/*/*ommunitheme
       rm -rf $SNAPCRAFT_PART_INSTALL/share/icons/Suru
+      rm -rf $SNAPCRAFT_PART_INSTALL/share/*/Yaru
```

Comment:
Do you mind future-proofing this with 'Yaru*' (so that dark variant is removed as well once available?)

Reply:
Done.